### DISCUSSION TITLE
Use with Hugo CMSes such as Forestry.io or Netflify CMS

### DISCUSSION THREAD

**Discussion by hughlilly** [thegetty/quire#197] [General]
To what extent could content-management systems for Hugo such as [Netlify CMS](https://www.netlifycms.org) or [Forestry.io](http://forestry.io) be used with Quire? I haven't used either and am new to SSGs, having done only a small amount of development work to build a Google Chrome extension in the past. I've worked a lot with Python and have more than a novice understanding of that, but don't have much JS knowledge; I know Markdown well enough though.

I'd be interested to know if it's possible to set up a CMS on top of Quire so that some of my less technically minded colleagues could collaborate on projects. We have existing content we'd like to port to Quire, and while I could do this by 'hand' using the Quire CLI, it would be cool to be able to present colleagues with a CMS, even if it were a bare-bones one. I suppose the difficulty would come in incorporating IIIF. (In an ideal world, such a CMS would eventually [connect to our API](http://aucklandmuseum.com/discover/collections/our-data) to pull collection images and metadata though.)

Thank you,
Hugh

**Comment by kristhayer11** [comment]
@hughlilly  We are just publishing a digital catalogue raissonné (later today) that uses an API to pull data from our DAMS (TMS and Mediabin) into Quire. Nearly all the catalogue entries are pulled (and reformatted) through the API before ingesting into Quire. A small number of customizations are handle with google sheets, but these, too, will eventually be handled by the API. Our challenge was that we did not want to alter the data standards in TMS, but the curator wanted the data presented in a particular way. The API accomplishes this and also dishes up IIIF image files. I am not familiar with Netlify CMS or Forestry.io. And I am not sure if our approach addresses any of your goals, but thought I'd share our experience. The url for our pub is https://artsmia.org/highpoint-editions; the Catalogue Raisonné section is the part that is pulling data from our DAMs.

**Reply by hughlilly** [reply]
Hi @kristhayer11; many thanks for this reply. Your publication looks great! Would you be able to expand on how you use Google Sheets? If you would prefer to email let me know and I will find a way to send you my email address. Thanks, Hugh.

**Reply by Erin-Cecele** [reply]
@kristhayer11 If email is preferred, let me know. I am more than happy to connect you and @hughlilly over email.

**Reply by kristhayer11** [reply]
@Erin-Cecele Good idea! Yes, please connect me with @hughlilly via email. I have to loop in @kjell also. Thank you!
